components/scripts/lib/git.sh

```diff
@@ -29,8 +29,8 @@ git_clone_project() {
    debug git clone --depth=1 ${git_options} ${branch} "${git_repo}" "${clone_dir}"
 
    # shellcheck disable=SC2086  # we want $git_options and $branch to expand into multiple arguments
-   git clone --depth=1 ${git_options} ${branch} "${git_repo}" "${clone_dir}" || die "ERROR: Unable to clone from ${git_repo}." 2
-   cd "${clone_dir}" || die "Unable to access git repository directory ${clone_dir}." 2
+   git clone --depth=1 ${git_options} ${branch} "${git_repo}" "${clone_dir}" || die "ERROR: Unable to clone from ${git_repo}."
```

Comment:
@erichaagdev this change (and similar changes) will likely create a merge conflict between our two PRs, but it should be fairly easy to resolve.